File: MODUL_3/C/PRAK304_2310817310014_MUHAMMADRIZKISAPUTRA.c
#include<stdio.h>

int main(){

int a;
printf("");
scanf("%d", &a);

if (a >=1 && a <=10){
    printf("satuan");

}   else if (a==0){
    printf("Nol");

}   else if (a >=100){
    printf("Anda mengimput Melebihi Limit Bilangan");

}   else if (a >=20 && a <= 99){
    printf("Puluhan");

}   else {
    printf("Belasan");

}

   return 0;
}

File: MODUL_3/Py/PRAK304_2310817310014_MuhammadRizkiSaputra.py
a = int(input())

if 1 <= a <= 10:
    print("satuan")
elif a == 0:
    print("Nol")
elif a >= 100:
    print("Anda menginput Melebihi Limit Bilangan")
elif 20 <= a <= 99:
    print("Puluhan")
else:
    print("Belasan")
